eZ/Bundle/EzPublishIOBundle/Resources/config/io.yml

```diff
@@ -1,3 +1,10 @@
+parameters:
+    ibexa.platform.io.nfs.adapter.config:
+        directory: '%dfs_nfs_path%/$var_dir$/$storage_dir$'
+        writeFlags: ~
+        linkHandling: ~
+        permissions: [ ]
```

Comment:
Exposed also in meta, for custom project configuration. Seems to me array is more readable than a parameter per setting.